tensorboard/webapp/metrics/views/card_renderer/image_card_component.ng.html

```diff
@@ -82,7 +82,7 @@
     <mat-slider
       class="step-slider"
       color="primary"
-      [ngClass]="[linkedTimeSelection && !linkedTimeSelection.clipped && linkedTimeSelection.endStep !== null ? 'hide-slider' : '']"
+      [ngClass]="[linkedTimeSelection && linkedTimeSelection.endStep !== null ? 'hide-slider' : '']"
```

Comment:
Because I am changing the definition of "clipped" the image card now needs to render the slider when the time selection is clipped.

**Question**: Should We hide the slider when the `timeSelection` is fully outside of the `minMaxStep` instead?

Reply:
I would prefer to use the double slider than single (i.e. still hide it). Although from UI perspective it's not making a lot of differences (the line is colored on single slider vs gray on double slider)
single 
=====o----
double
-------o----

Reply:
At the moment I'm not actually changing the behavior here. I personally don't have an opinion on the slider but I would be happy to change it in a future PR.

For reference:
Nothing Enabled (Also looks the same with just Step Selection enabled)
Before:
![image](https://user-images.githubusercontent.com/78179109/193695567-41ea788d-59f0-4d17-a139-ae80a021793c.png)

After:
![image](https://user-images.githubusercontent.com/78179109/193695586-af67c683-f9a0-4f7c-85a0-d21dcc81afed.png)

Linked Time Enabled
Before:
![983f8d79-70d3-4e84-8965-c766ee28cceb](https://user-images.githubusercontent.com/78179109/193695775-7e291a7a-52b6-4d8f-8775-61e3754647b2.gif)

After:
![e46aaa25-b065-49c2-aa71-90f9f41c0094](https://user-images.githubusercontent.com/78179109/193695843-d736174f-49c5-43d5-a476-e22f54b1d0bc.gif)